### ISSUE TITLE
Authorship Criteria

### ISSUE THREAD

**Issue by certik** [sympy/sympy-paper#195]
Dear co-authors (@asmeurer, @smichr, @mattpap, @certik, @skirpichev, @mrocklin, @aktech, @scolobb, @moorepants, @leosartaj, @thilinarmtb, @flacjacket, @ellisonbg, @rpmuller, @Upabjojr, @hargup, @shivamvats, @fredrik-johansson, @fabianp, @mattcurry, @aterrel, @rouckas, @ashutoshsaboo, @isuruf, @Sumith1896, @rc, @scopatz), here are the ICMJE [authorship criteria](http://www.icmje.org/recommendations/browse/roles-and-responsibilities/defining-the-role-of-authors-and-contributors.html#two) that we follow:
1. Substantial contributions to the conception or design of the work; or the
   acquisition, analysis, or interpretation of data for the work; AND
2. Drafting the work or revising it critically for important intellectual
   content; AND
3. Final approval of the version to be published; AND
4. Agreement to be accountable for all aspects of the work in ensuring that
   questions related to the accuracy or integrity of any part of the work are
   appropriately investigated and resolved.

I have verified that all of you satisfy 1. and 2. (and the info is easily accessible in [authors.md](https://github.com/sympy/sympy-paper/blob/9fb21e871bcfa5d0ff3c8dfb70749e7473d5b6d3/authors/authors.md) for anyone to browse). The version to be published is 9fb21e871bcfa5d0ff3c8dfb70749e7473d5b6d3, can you all give a final approval of this version (thus satisfying 3.) and also agree to be accountable for all aspects of the work in ensuring that questions related to the accuracy or integrity of any part of the work are appropriately investigated and resolved (thus satisfying 4.).

Just post a comment that you agree. If you do not agree to 3. or 4., let me know.

These people agreed to 3. and 4. below in a comment:
- [x] @asmeurer
- [x] @smichr
- [x] @mattpap
- [x] @certik
- [x] @skirpichev
- [x] @mrocklin
- [x] @aktech
- [x] @scolobb
- [x] @moorepants
- [x] @leosartaj
- [x] @thilinarmtb
- [x] @flacjacket
- [x] @ellisonbg
- [x] @rpmuller
- [x] @Upabjojr
- [x] @hargup
- [x] @shivamvats
- [x] @fredrik-johansson
- [x] @fabianp
- [x] @mattcurry
- [x] @aterrel
- [x] @rouckas
- [x] @ashutoshsaboo
- [x] @isuruf
- [x] @Sumith1896
- [x] @rc
- [x] @scopatz

**Comment by certik** [comment]
For myself, I approve the version and agree to 4.

**Comment by asmeurer** [comment]
I approve the version and agree to 4.

**Comment by rouckas** [comment]
I approve the final version and agree to 4.

**Comment by moorepants** [comment]
Have we already submitted to PeerJ and been accepted without revisions? Or is this a request for what version will be submitted to PeerJ?

**Comment by rpmuller** [comment]
I approve the final version and agree to 4.

**Comment by sumith1896** [comment]
I approve the version and agree to 4.

**Comment by scolobb** [comment]
I approve the version and agree to 4.

**Comment by flacjacket** [comment]
I approve the version and agree to 4.

**Comment by shivamvats** [comment]
I approve the final version and agree to 4.

**Comment by mattcurry** [comment]
I approve this version and agree to 4.

**Comment by certik** [comment]
@moorepants we submitted to PeerJ together with SIAM, but we didn't get an agreement with each co-author about 3. and 4. These are our authorship criteria, it's up to us when we do it. Now is a good time to do it, since the revision above is our final version, that we will submit to PeerJ.

**Comment by rc** [comment]
I approve this version and agree to 4.

**Comment by leosartaj** [comment]
I approve the version and agree to 4.

**Comment by aktech** [comment]
I approve the version and agree to 4.

**Comment by thilinarmtb** [comment]
I approve the version and agree to 4.

**Comment by isuruf** [comment]
I approve the version and agree to 4.

**Comment by fabianp** [comment]
I approve the version and agree to 4

**Comment by rpmuller** [comment]
I approve the version and agree to 4.

**Comment by ashutoshsaboo** [comment]
I approve the version and agree to 4. :+1:

**Comment by fredrik-johansson** [comment]
I approve the version and agree to 4.

**Comment by Upabjojr** [comment]
I approve the version and agree to 4.

**Comment by hargup** [comment]
I too approve the version and agree to 4.

**Comment by aterrel** [comment]
I approve the version and agree to 4

**Comment by mattpap** [comment]
I approve the version and agree to 4.

**Comment by certik** [comment]
@skirpichev wrote me in private email:

> Yeah, I approve this version and agree to 4.
> 
> Feel free to quote me in that thread.

**Comment by smichr** [comment]
I approve the version and agree to 4.

**Comment by scopatz** [comment]
approve

**Comment by certik** [comment]
@scopatz, to be clear, do you also agree to do 4.?

**Comment by scopatz** [comment]
explicitly, yes

**Comment by mrocklin** [comment]
Approve (3) and agree (4)

**Comment by ellisonbg** [comment]
I approve of 3 and agree to 4.

**Comment by moorepants** [comment]
I agree to submit the current version to PeerJ. I'll approve the final submission after it is reviewed and we've completed the revisions. I don't think we can approve #3 until we have the final version in hand.

**Comment by certik** [comment]
@moorepants just to be on the safe side, are you giving your approval to the version 9fb21e871bcfa5d0ff3c8dfb70749e7473d5b6d3, as indicated in the issue above? If there are further revisions to be needed, we will approve those later.

Also, do you agree to do 4.?

**Comment by moorepants** [comment]
Yes, I agree to the version you posted and to 4.
